documentation/ci-fix.md

```diff
@@ -44,3 +44,22 @@ oav validate-spec <swagger-spec-path>
 Please see [readme](https://github.com/Azure/oav/blob/master/README.md) for how to install or run tool in details.
 Or you can run it in [OpenAPI Hub](https://portal.azure-devex-tools.com/tools/static-validation/static/errors/default)
 Refer to [Semantic and Model Violations Reference](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/Semantic-and-Model-Violations-Reference.md) for detailed description of validations and how-to-fix guidance.
+
+## Avocado
+
+Run avocado locally:
+
+```
+npm install -g @azure/avocado
+
+avocado
+```
+
+When type avocado in command line, avocado will validate in the current directory.
+
+NOTE: When running in azure devops Avocado only report new errors involved in PR, but ignore the previous existing errors. So when running in local machine, Avocado report all errors.
```

Comment:
The NOTE is a little bit confusing. Some tuning based my understanding...

Note: when running in Azure Devops (what do you referring to? Do you mean PR pipeline? ), Avocado only reports errors with file updates in the PR, but ignore errors existing in the base. However, when running Avacado against local directory, it reports all errors existing in the files.

Reply:
Yes. It's Azure pipeline. updated the note.